spec/controllers/dishes_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe DishesController, type: :controller do
+  let(:user)    { FactoryBot.create(:user) }
+  let(:admin)   { FactoryBot.create(:user, :admin) }
```

Comment:
Check this out!
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#rspec

if you add a single line to your rspec config, you can get rid of `FactoryBot.` and just use `create(:user)` etc.